guides/micronaut-mqtt/app/src/main/resources/application.yml

```diff
@@ -0,0 +1,6 @@
+#tag::mqtt[]
+mqtt:
+  client:
+    server-uri: tcp://localhost:1883
+    client-id: ${random.uuid}
```

Comment:
should `${random.uuid}` be `"micronaut-cli"` to agree with the CLI app?

Reply:
if not this looks weird because it's the same as what's generated in the initial app

Reply:
I set both to be `client-id: ${random.uuid}` which is what we generate in launch by default.